pkg/crinit/aggregated/aggregated.go

```diff
@@ -113,10 +113,16 @@ func NewCmdAggregated(cmdOut io.Writer, pathOptions *clientcmd.PathOptions, defa
 	opts := &aggregatedClusterRegistryOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "aggregated CLUSTER_REGISTRY_NAME --host-cluster-context=HOST_CONTEXT",
-		Short:   "Initializes an aggregated cluster registry",
-		Long:    longCommandDescription,
-		Example: commandExample,
+		Use:   "aggregated",
+		Short: "Subcommands to manage an aggregated cluster registry",
+		Long:  "Commands used to manage an aggregated cluster registry. That is, a cluster registry that is aggregated with another Kubernetes API server.",
+	}
+
+	initCmd := &cobra.Command{
+		Use:     "init",
```

Comment:
Shouldn't this be `"init  CLUSTER_REGISTRY_NAME --host-cluster-context=HOST_CONTEXT"`?